core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/configure-k8s-apps-monitoring.yml

```
@@ -1,16 +1,33 @@
 ---
-- name: Set master hostname variable
-  set_fact:
-    master_hostname: "{{ groups['kubernetes_master'] | first }}"
+- name: Set facts for "classic epiphany k8s"
+  when: k8s_as_cloud_service is defined and k8s_as_cloud_service is false
+  block:
+    - name: Set master hostname variable
+      set_fact:
+        master_hostname: "{{ groups['kubernetes_master'] | first }}"
 
-- name: Set api server address variable
-  set_fact:
-    api_server_address: "https://{{ master_hostname }}:6443"
+    - name: Set api server address variable
+      set_fact:
+        api_server_address: "https://{{ master_hostname }}:6443"
+
+    - name: Set Kubernetes credentials
+      import_role:
+        name: kubernetes_master
+        tasks_from: copy-kubeconfig
+       
+- name: Set facts for "k8s as cloud service"
+  when: k8s_as_cloud_service is defined and k8s_as_cloud_service is false
```

Wait, should this be negated here or not? What am I missing? :thinking: 
```suggestion
  when: k8s_as_cloud_service is defined and k8s_as_cloud_service
```